routes/api/follows.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const router = express.Router();
+const createError = require('http-errors');
+const { sequelize, Follow } = require("../../db/models");
+const {asyncHandler} = require('../utils');
+
+/* Delete a follower from a user */
+router.delete('/follows/:id', asyncHandler(async (req, res) => {
+
+        /* Initialize follow object */
+        const follow = await Follow.findPk(req.params.id);
+
+        /* Return error 404, if 
+           there is no follow object */
+        if (typeof follow === "undefined" ) {
+            return next(createError(404)); 
+        };
+
+        if (!res.locals.authenticated || res.locals.user.id !== follow.followerId) {
+            /* Return error 401 if
+                   user is not authenticated, or
+                   user is not the follower */
+            return next(createError(401));
```

Comment:
The code does not have access to a `next` in this scope

Reply:
`next` can now be accessed.